skada/deep/modules.py

```diff
@@ -110,5 +129,32 @@ def forward(self, x, alpha=None):
         alpha: float
             Parameter for the reverse layer.
         """
-        reverse_x = GradientReversalLayer.apply(x, alpha)
-        return self.classifier(reverse_x)
+        reverse_x = GradientReversalLayer.apply(x, self.alpha)
+        return self.classifier(reverse_x).flatten()
+
+
+class MNISTtoUSPSNet(nn.Module):
+    def __init__(self):
+        super(MNISTtoUSPSNet, self).__init__()
+        self.conv1 = nn.Conv2d(1, 32, 3, 1)
+        self.conv2 = nn.Conv2d(32, 64, 3, 1)
+        self.dropout1 = nn.Dropout(0.25)
+        self.dropout2 = nn.Dropout(0.5)
+        self.fc1 = nn.Linear(9216, 128)
+        self.fc2 = nn.Linear(128, 10)
+        self.relu = nn.ReLU()
```

Comment:
use different relu (self.reu1, self.relu2 ...) for erach layer so that you can select the embedding as the output or a relu